File: src/main/java/top/zbeboy/isy/web/bean/data/organize/OrganizeBean.java
package top.zbeboy.isy.web.bean.data.organize;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.Organize;

/**
 * Created by lenovo on 2016-09-25.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class OrganizeBean extends Organize {
    private String schoolName;
    private String collegeName;
    private String departmentName;
    private String scienceName;
    private int schoolId;
    private int collegeId;
    private int departmentId;
}

File: src/main/java/top/zbeboy/isy/web/bean/data/organize/OrganizeBean.kt
package top.zbeboy.isy.web.bean.data.organize

import top.zbeboy.isy.domain.tables.pojos.Organize

/**
 * Created by zbeboy 2017-12-03 .
 **/
class OrganizeBean: Organize() {
    var schoolName: String? = null
    var collegeName: String? = null
    var departmentName: String? = null
    var scienceName: String? = null
    var schoolId: Int = 0
    var collegeId: Int = 0
    var departmentId: Int = 0
}